src/reference/asciidoc/jdbc.adoc

```diff
@@ -632,6 +632,14 @@ Such connection pools do normally expect that issued connections are closed with
 For this need of an exclusive connection, it is also recommended that a JVM only runs a single `PostgresChannelMessageTableSubscriber` which can be used to register any number of subscriptions.
 ====
 
+==== Transaction support
+Specifying a `PlatformTransactionManager` on `PostgresSubscribableChannel` will notify subscribers in a transaction.
```

Comment:
We also need to mention that this feature is available `starting with version 6.0.5`

Reply:
done